libs/perun/components/src/lib/consents-list/consents-list.component.html

```diff
@@ -0,0 +1,126 @@
+    [tableId]="tableId"
+    [dataLength]="dataSource.filteredData.length"
+    [pageSizeOptions]="pageSizeOptions">
+    <table
+      mat-table
+      [dataSource]="dataSource"
+      matSort
+      matSortActive="status"
+      matSortDirection="asc"
+      matSortDisableClear
+      multiTemplateDataRows
+      class="w-100">
+      <ng-container matColumnDef="select">
+        <th mat-header-cell *matHeaderCellDef class="align-checkbox">
+          <mat-checkbox
+            color="primary"
+            (change)="$event ? masterToggle() : null"
+            [checked]="selection.hasValue() && isAllSelected()"
+            [indeterminate]="selection.hasValue() && !isAllSelected()">
+          </mat-checkbox>
+        </th>
+        <td mat-cell *matCellDef="let consent" class="static-column-size align-checkbox">
+          <mat-checkbox
+            color="primary"
+            (click)="$event.stopPropagation()"
+            (change)="$event ? selection.toggle(consent) : null"
+            [checked]="selection.isSelected(consent)">
+          </mat-checkbox>
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="status">
+        <th *matHeaderCellDef mat-header-cell mat-sort-header>
+          {{'SHARED_LIB.PERUN.COMPONENTS.CONSENTS_LIST.STATUS' | customTranslate | translate}}
+        </th>
+        <td mat-cell *matCellDef="let consent">
+          <perun-web-apps-consent-status
+            [consentStatus]="consent.status"></perun-web-apps-consent-status>
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="name">
+        <th *matHeaderCellDef mat-header-cell mat-sort-header>
+          {{'SHARED_LIB.PERUN.COMPONENTS.CONSENTS_LIST.SERVICE_NAME' | customTranslate | translate}}
+        </th>
+        <td mat-cell *matCellDef="let consent">
+          {{consent.consentHub.name}}
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="expandedDetail">
+        <td mat-cell *matCellDef="let consent" [attr.colspan]="displayedColumns.length">
+          <div
+            [@detailExpand]="consent === expandedConsent ? 'expanded' : 'collapsed'"
+            class="consent-detail">
+            <div class="p-3">
+              <div *ngIf="consent.status === 'GRANTED'">
+                {{'SHARED_LIB.PERUN.COMPONENTS.CONSENTS_LIST.GRANTED_TEXT' | customTranslate | translate}}
+                :
+              </div>
+              <div *ngIf="consent.status === 'REVOKED'">
+                {{'SHARED_LIB.PERUN.COMPONENTS.CONSENTS_LIST.REVOKED_TEXT' | customTranslate | translate}}
+                :
+              </div>
+              <div *ngIf="consent.status === 'UNSIGNED'">
+                {{'SHARED_LIB.PERUN.COMPONENTS.CONSENTS_LIST.UNSIGNED_TEXT' | customTranslate | translate}}
+                :
+              </div>
+              <ul>
+                <li *ngFor="let attribute of consent.attributes">
+                  {{ attribute.displayName }}
+                </li>
```

Comment:
This seems like a duplication of what is already in the `consent-request`. Perhaps we could have a component for this.

Reply:
its a little bit of duplication - but the duplication is only the list of attributes and the text - and that's it - the buttons have different CSS